app/models/organization.rb

```diff
@@ -2,4 +2,5 @@ class Organization < ApplicationRecord
   validates :name, presence: true
   has_many :needs, dependent: :destroy
   has_many :surplus, dependent: :destroy
+  has_many :claims, dependent: :destroy
```

Comment:
Since claim also belongs to surplus. perhaps it should not be destroyed if an organization is destroyed.